.azure-pipelines-templates/deploy_aci.yml

```diff
@@ -122,6 +123,7 @@ jobs:
                 --count ${{ parameters.secondaries.count }} \
                 --aci-file-share-name ccfcishare \
                 --aci-file-share-account-name ccfcistorage \
+                --ports 22 8000 8001 \
```

Comment:
`8000` doesn't seem to be used in the Python infra, which makes me think that new secondary node probably doesn't have have a working node-to-node channel with the primary node. This should be confirmed what calling [`GET /node/commit`](https://microsoft.github.io/CCF/main/operations/operator_rpc_api.html#get--node-commit) on the new node. I expect it to be stuck at 0?

To fix this, the primary node should bind to `0.0.0.0:8000` (published address: `<primary_address>:8000`) and the target address for the new secondary node should be `<primary_address>:8000`. I don't think we have a practical way to set the node-to-node address configuration for nodes just yet, so that'll probably need to be added to the infra.

Reply:
Don't we use it in the AciShim?